### ISSUE TITLE
Installing refurb (on conda) requires downgrading mypy

### ISSUE THREAD

**Issue by oradwastaken** [dosisod/refurb#257]
I don't know if mypy was unintentionally pinned, but when trying to install refurb using conda, I'm forced to downgrade it to a (much) older version:

`The following packages will be DOWNGRADED:

  mypy                                1.4.1-py311heffc1b2_0 --> 0.991-py311he2be06e_0 `

Any chance we can "refurbish" the version of mypy in the package requirements?

**Comment by dosisod** [comment]
Thank you for bringing this to my attention! I've never used Conda, so it's really cool to see so many people downloading Refurb through Conda.

After doing some digging it appears the `index.json` file is out of date, which is why you are getting the old version:

```json
{
  "arch": null,
  "build": "pyhd8ed1ab_0",
  "build_number": 0,
  "depends": [
    "mypy >=0.981,<=0.991",
    "python >=3.10,<4.0",
    "tomli >=2.0.1,<3.0.0"
  ],
  "license": "GPL-3.0-or-later",
  "name": "refurb",
  "noarch": "python",
  "platform": null,
  "subdir": "noarch",
  "timestamp": 1687882062166,
  "version": "1.17.0"
}
```

If you look at the Mypy version in the `pyproject.toml` file, it should be `>=0.981`:

https://github.com/dosisod/refurb/blob/45d8e58df76976d8f954b1aac83ddda0e08da9d6/pyproject.toml#L18-L21

I'm new to Conda, so I don't know how to go about updating this package. It would appear someone (graciously) uploaded it for me, though I don't know where to go or who to ask to update this. I'll look into this and keep you posted.

Thank you for reporting this!

**Comment by oradwastaken** [comment]
Hey cool, a response! Thanks for getting back to me so quickly!

I have no idea how to update/push packages to conda. I looked into it briefly once and it seemed like more trouble than it was worth... but it sounds like if you really want to maintain a popular package you might have to extend yourself to that domain too.

Thanks again!

**Comment by dosisod** [comment]
@oradwastaken I figured out how to update the package. I just opened a PR for it, so hopefully it will be fixed relatively soon!

If you're curious how to update a Conda package:

* Go to `https://github.com/conda-forge/PACKAGE-feedstock` (replacing `PACKAGE` with the package you want to update)
* Fork it, make a branch
* Commit your changes
* Open a PR

Took me a while to figure that out, hope it helps!
